MekHQ/unittests/mekhq/gui/enums/MHQTabTypeTest.java

```diff
@@ -190,62 +189,6 @@ public void testIsStratCon() {
     }
     //endregion Boolean Comparison Methods
 
-    /**
-     * This test is disabled because it will run through all the GUI tabs and initialize them, which
-     * is not a quick process. This also requires lots more mock handling to actually go through
-     * the initializations too.
-     */
-    @Disabled
-    @Test
-    public void testCreateTab() {
```

Comment:
This is disabled and I remember there had been discussion about it. Dont know if it should be removed though.